### PR TITLE
Update loading state to show spinner and translucent background over chart

### LINKED ISSUE #502: The loading experience is quite 'jarring'
When you switch instruments or time period all of the charts disappear, being replaced with a 'loading ...' text. This is visually quite jarring.

Could it perhaps render the charts (potentially without data) but at a reduced opacity?

Might be worth asking Rui or one of our other UXers for ideas.

### PR DESCRIPTION
**Pull request by jleft**
Fixes #502

recording

### CODE REVIEW

File: src/assets/js/app.js [hunk 1]
```diff
@@ -232,12 +232,12 @@ export default function() {
     }
 
     function loading(isLoading, error) {
+        var spinner = '<div class="spinner"></div>';
```
Comment (djmiley) [inline src/assets/js/app.js]: Do we need both declarations of spinner html?

File: src/assets/js/app.js [hunk 2]
```diff
@@ -18,6 +18,9 @@ export default function() {
 
     var appTemplate = '<div class="container-fluid"> \
         <div id="notifications"></div> \
+        <div id="loading-status-message"> \
+            <div class="spinner"></div> \
```
Comment (djmiley) [inline src/assets/js/app.js]: Do we need both declarations of spinner html?

Comment (djmiley) [inline src/assets/js/app.js]: As in not totally sure line 22 is needed.

**Comment by djmiley** [comment]
Looks good. :)

**Comment by jdhodges431** [comment]
:+1:

File: src/assets/js/app.js (hunk 2 above)
Comment (jleft) [inline src/assets/js/app.js]: Removed :)

**Comment by djmiley** [comment]
:+1:

**Pull request merged**